js/webviewMenu.js

```diff
@@ -194,6 +194,18 @@ const webviewMenu = {
     }
 
     if (link || (mediaURL && !mediaURL.startsWith('blob:'))) {
+      if (link.startsWith('mailto:')) {
+        clipboardActions.push({
+          label: l('copyEmailAddress'),
+          click: function () {
+            var ematch = link.match(/(?<=mailto:)[^\?]+/)
+            if (ematch) {
```

Comment:
Minor issue: `ematch` should probably be found before the item is created, and then if there isn't a match, the menu item shouldn't be shown at all.